udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -402,16 +402,13 @@ <h2 id="community-reuses" class="fr-h2">{{ _('Reuses') }} <sup>{{ reuses|length
         </div>
         {% endif %}
         {% endblock %}
-
-        <h2 id="community-discussions" class="fr-h2">{{ _('Discussions') }} <sup class="discussions-count">0</sup></h2>
-        <div class="row">
-            <div class="col-4 col-md-12 text-grey-300">
-                <p>{{ _('Discussion between the organization and the community about this dataset.') }}</p>
-            </div>
-        </div>
         <div class="vuejs">
-            <discussion-threads ref="discussions" subject-id="{{ dataset.id }}"
-                subject-class="{{ dataset.__class__.__name__ }}">
+            <discussion-threads
+                ref="discussions"
+                subject-id="{{ dataset.id }}"
+                subject-class="{{ dataset.__class__.__name__ }}"
+            >
+                <p>{{ _('Discussion between the organization and the community about this dataset.') }}</p>
```

Comment:
Maybe change wording to `producer` or make it conditional to include user as well?

Reply:
Could you also update the wording accordingly here (and change `dataset` by `reuse`)? https://github.com/etalab/udata-front/blob/2054d776d3bd6f3901497e354cc7d75233733cc0/udata_front/theme/gouvfr/templates/reuse/display.html#L241 :pray: